transport_nantes/asso_tn/templates/asso_tn/base_mobilitain.html

```diff
@@ -103,11 +112,27 @@
 			if they are designated with a string.
 			{% endcomment %}
 			{% if is_static %}
-				<meta name="og:image" content="{% static social.og_image %}" />
+				{# OG image #}
+				{% if social.og_image and social.og_image != "velopolitain/v1.png" %}
+					<meta property="og:image" content="{{social.og_image.url}}" />
+				{% elif social.twitter_image and social.twitter_image != "asso_tn/mobilite-pour-tous.jpg"%}
+					<meta property="og:image" content="{{ social.twitter_image.url }}" />
+				{% elif page.header_image  %}
+					<meta property="og:image" content="{{page.header_image.url}}" />
+				{% else %}
```

Comment:
You seem confident that social.og_image_url will be defined
That confidence merits a comment explaining itself.

Reply:
I think the default will be always there because when i try empty the default context replace everythings (og image,twitter_image,twitter_title and twitter_description) that is  empty.